source/features/sort-issues-by-update-time.js

```diff
@@ -0,0 +1,41 @@
+import select from 'select-dom';
+import {getUsername} from '../libs/utils';
+
+function getDefaultQuery(link) {
+	// Query-less URLs imply some queries.
+	// When we explicitly set ?q=* they're overridden,
+	// so they need to be manually added again.
+	const queries = [];
+
+	// Repo example: is:issue is:open
+	queries.push(/\/pulls\/?$/.test(link.pathname) ? 'is:pr' : 'is:issue');
+	queries.push('is:open');
+
+	// Header nav example: is:open is:issue author:you archived:false
+	if (link.pathname === '/issues' || link.pathname === '/pulls') {
+		queries.push(`author:${getUsername()}`);
+		queries.push('archived:false');
```

Comment:
@bfred-it could you add this behind a version check? I have to remove it every time I use those `https://github.myenterprise.io/pulls`, which is stuck at 2.11 at the moment